src/modules/account/components/Accounts/Accounts.js

```diff
@@ -1,33 +1,30 @@
 import React, { useEffect } from 'react';
-
```

Comment:
NRTTL:
My account has bot LSK and ENVT. 

When i am on LSK chain
<img width="567" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/865fb8ef-ce6f-4278-a82c-db4fcd9df601">

When i am on Enevti i see both
<img width="472" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/8fd560bc-9076-4266-8be7-aa0f7ca0436f">

Is this intended behaviour? Shouldnt i see both in each case?